autogen/docs/codetab.md.mustache

```diff
@@ -5,6 +5,17 @@ to use immediately go in the Command Center; commands you want to save and use
 later, over and over again, are found in the Code tab.
 ![[codetab.gif]]
 
+The Code tab can be opened in a separate window from the main NetLogo window. It can then be moved and resized.
+
+### Ways to separate and rejoin the Code tab
+- The "Code Tab in separate window" checkbox can be used to switch between the joined (part of the main NetLogo window) and separated states.
```

Comment:
It seems odd to have the parenthetical here, maybe just "...can be used to switch between joined to the main NetLogo window and separated into its own window..."